diff --git a/snippets/drupal/checkbox.snippet b/snippets/drupal/checkbox.snippet
index 36a3c54..3638641 100644
--- a/snippets/drupal/checkbox.snippet
+++ b/snippets/drupal/checkbox.snippet
@@ -1,4 +1,4 @@
-$form['${1:/* form element */'] = array(
+$form['${1:/* form element */}'] = array(
 	'#type' => 'checkbox', 
 	'#title' => t('${2:/* title */}'),
 	'#description' => t('${3:/* description */}'),
diff --git a/snippets/drupal/checkboxes.snippet b/snippets/drupal/checkboxes.snippet
index d1129ae..c788fb9 100644
--- a/snippets/drupal/checkboxes.snippet
+++ b/snippets/drupal/checkboxes.snippet
@@ -1,8 +1,8 @@
-$form['${1:/* form element */'] = array(
+$form['${1:/* form element */}'] = array(
 	'#type' => 'checkboxes', 
 	'#title' => t('${2:/* title */}'),
 	'#description' => t('${3:/* description */}'),
 	'#options' => array(
-		'${4:/* key */}' => t('${5:/* value */}),
+		'${4:/* key */}' => t('${5:/* value */}'),
 	);
 );
